mods/d2k/sequences/infantry.yaml

```diff
@@ -35,24 +43,24 @@ light_inf:
 		Facings: -8
 		Transpose: true
 	die1: DATA.R8
-		Frames: 382, 389, 396, 403, 410, 417, 424, 431, 438, 439, 440, 441
+		Frames: 382, 389, 396, 403, 410, 417, 424, 431, 438, 443, 448, 453
```

Comment:
There isn't really a pattern here. For example the rocket trooper death animation has a 1 step death decaying animation https://github.com/OpenRA/OpenRA/pull/11949/files#diff-6211bf63a321e3de7a22a5294c07c2e5R114 which we already set up.